File: prob4/prob4.rb
def rand_print(num)
	(0..num).each do |x|
		if x % 2 == 0
			puts x
		end
	end
end

rand_num = rand(1..100)
puts "The number is " + rand_num.to_s + "\n\n"
rand_print(rand_num)

File: prob4/prob4.js
var rand_print = function (rand_num){
	for (var i = 0; i <= rand_num; i++) {
		if (i % 2 == 0){
			document.write(i + ", ")
		}
	}
}

var rand_num = (Math.floor((Math.random() * 100)))
document.write("The number is " + rand_num + "<br/>")
rand_print(rand_num)

